### ISSUE TITLE
file dialog: explicitely set View to "Short View" (since tree view apparently is new default)

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-issues-old#185]
![Screenshot_20190409_122806](https://user-images.githubusercontent.com/840070/55793661-c3b43300-5ac3-11e9-91a4-801d024dd240.png)

**Comment by NuLogicSystems** [comment]
On my fresh install launching "open file" from Firefox, this is already set to "short view".
Update, I just tried kate and the same thing,  short view is the default.

**Comment by star-buck** [comment]
It will change in the future, so lets explicitely set it!

**Comment by NuLogicSystems** [comment]
Already did. ;)

**Comment by star-buck** [comment]
@llelectronics "ready on git" is for both versions, i just have to check it actually "lands" on the respective ISOs/imgs?

**Comment by llelectronics** [comment]
Yes for both. We use the same source for Rolling and Desktop here. So the change landed on dci already for example and should be in the next Netrunner Desktop release.
